crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -1,5 +1,57 @@
 #![deny(missing_docs)]
-//! This module is used to configure the conductor
+//! This module is used to configure the conductor.
+//!
+//! #### Example minimum conductor config:
+//!
+//! ```rust
+//! let yaml = r#"---
+//!
+//! ## Configure the keystore to be used.
+//! keystore:
+//!
+//!   ## use an in-process keystore with default database location.
+//!   type: lair_server_in_proc
+//!
+//! ## Configure an admin websocket interface at a specific port.
+//! admin_interfaces:
+//!   - driver:
+//!       type: websocket
+//!       port: 1234
+//!       allowed_origins: "*"
+//!
+//! ## Configure the network.
+//! network:
+//!
+//!   ## Use the holo-provided default production bootstrap server.
+//!   bootstrap_service: https://bootstrap.holo.host
+//!
+//!   ## [deprecated] Please just include this in your config file for now.
```

Comment:
@jost-s notes that this isn't deprecated and is still required.

Reply:
```suggestion
//!   ## This currently has no effect on functionality but is required. Please just include as-is for now.
```

Reply:
Let me know if that change works : )